docs/migrate/from-hono.md

```
@@ -1062,7 +1062,7 @@ import { Elysia } from 'elysia'
 
 const app = new Elysia({
 	cookie: {
-		secret: 'secret'
+		secret: 'secret' // Need to set to have cookies signed
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)